src/rivescript.coffee

```diff
@@ -116,6 +117,13 @@ class RiveScript
     return VERSION
 
   ##
+  # Promise Promise
+  #
+  # Alias for RSVP.Promise
+  ##
```

Comment:
Add an example, either here or under replyAsync, showing how to use `rs.Promise` so that people writing asynchronous subroutines know to use this rather than try importing their own promise library and potentially having headaches figuring it out.